content/services/former_POWERdev_projects.rst

```diff
@@ -0,0 +1,22 @@
+Formerly Hosted Projects
+========================
+:slug: services/powerdev/former-projects
+:title: Formerly Hosted Projects
+:menu: POWERLinux/OpenPOWER Development Hosting, Former Projects, 2
+
+Below are a list of projects formerly hosted on OpenPOWER and former OpenPOWER
```

Comment:
I suggest we say "OpenPOWER _infrastructure at OSL_" and exclude the "former OpenPOWER partners". 
@ramereth thoughts?